prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,16 @@
+var languagesArray = ["html", "css", "javascript", "react", "node.js",]
+var thirdElement = languagesArray[2];
+var arrayLength = languagesArray;
+var lastItem = arrayLength[5];
+
+languagesArray.length;
+languagesArray.splice (5,0, "c#", "php",);
+languagesArray.pop ();
+languagesArray.unshift ("c++",);
+languagesArray.shift();
```

Comment:
You don't have to declare all of your variables at the top, try working through the steps in the instructions one more time, in order. If you are unclear on something, it's super-duper important that you send your instructor a slack message!